inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -1,8 +1,20 @@
 """Class for the coils creation."""
-from inductiva.scenarios import Scenario
 import math
 import random
 import numpy as np
+import os
+
+from typing import Optional
+from uuid import UUID
+
+from functools import singledispatchmethod
+from inductiva import tasks
+from inductiva.stellarators.simulators import Simsopt
```

Comment:
Please fix the imports to follow Inductiva's guidelines. Check Hugo's email.
You should not directly import classes.
Remember that note I sent about readability?